resolwe/storage/views.py

```diff
@@ -43,6 +44,56 @@ class UploadConfigSerializer(serializers.Serializer):
     config = serializers.DictField()
 
 
+class UploadCredentialsSerializer(serializers.Serializer):
+    """Serializer for upload credentials."""
+
+    AccessKeyId = serializers.CharField()
+    SecretAccessKey = serializers.CharField()
+    Token = serializers.CharField()
+    Expiration = serializers.DateTimeField()
+
+
+class UploadCredentials(ViewSet):
+    """Get the upload credentials."""
+
+    serializer_class = UploadCredentialsSerializer
+
+    def list(self, request):
+        """Return the JSON representing the upload configuration.
+
+        The returning object is JSON representation of the dictionary with the
```

Comment:
Only `credentials` dictionary is returned.

Reply:
Ah, yes, wrong description. And not exactly `credentials` dictionary, as it is a little different. Note `SessionToken` key.